.github/workflows/initializr-build.yml

```diff
@@ -44,6 +44,12 @@ jobs:
   validate-cas-overlay:
     needs: [build]
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        cfg:
+          - { casVersion: 6.3.3, bootVersion: 2.3.7.RELEASE }
+          - { casVersion: 6.4.0-RC2, bootVersion: 2.4.3 }
```

Comment:
Could we dynamically determine these somehow? or via gradle.properties? For the next version and when things change, one would have to remember to manually update this file too. There should be a way to update a version in one area and have the build pick it up, specially if that version is under dev. (Ignoring those that are released and frozen, like 6.3.3).

Perhaps one possibility would be to define these in the gradle.properties file somewhere, then create a gradle tasks that outputs these (as JSON, etc). Then, use the gradle task to create input/output variables for the build. Sort of like the main CAS build when it generates test categories.